cookbook/console/console_command.rst

```diff
@@ -82,6 +82,11 @@ for details.
 Getting Services from the Service Container
 -------------------------------------------
 
+.. caution::
+
+    The "container scopes" concept explained in this section has been deprecated
+    in Symfony 2.8 and it will be removed in Symfony 3.0.
```

Comment:
This section should actually be rewritten already, because the Translator has been refactored compared to what is described in this section. And IIRC, there is no issue with the Translator anymore (it will just be using your default locale)